### DISCUSSION TITLE
[rule]: padding-line-between-statements

### DISCUSSION THREAD

**Discussion by artola** [biomejs/biome#4380] [Ideas]
It would be nice to have this rule [`padding-line-between-statements`](https://eslint.org/docs/latest/rules/padding-line-between-statements).
May be I am wrong and there is some "similar" rule for handling this style issue.

**Comment by ematipico** [comment]
We don't intend to add stylistic rules in Biome linter.

Feel free to pitch this idea for the formatter, but check our philosophy first
